src/utils/global.ts

```diff
@@ -86,6 +86,17 @@ export class GlobalState {
 	windowSize: number = WindowSize.Large;
 
 	async init() {
+		await this.bootstrap();
+
+		// IMPORTANT: If on the /access-token/ page, automatically consent to continue loading. This is only
+		// done on OSS.
+		if (
+			window.location.pathname.startsWith('/access-token/') &&
+			this.bootstrapData.cluster == Rivet.cloud.BootstrapCluster.Oss
+		) {
+			global.grantConsent();
```

Comment:
@NathanFlurry this is a good question - related to #12, from a legal standpoint, can we do that? :P

Reply:
We discussed the legality of this in person, given that its OSS software running on your own machine its ok to do